### DISCUSSION TITLE
MonitorControl no longer opens

### DISCUSSION THREAD

**Discussion by agoldh20** [MonitorControl/MonitorControl#1401] [General]
I'm not sure what happened, but I was playing with the settings for when my computer wakes up because I was having issues with one monitor and now the application no longer opens. Not sure if I have a task hanging that I can restart? I did delete the app and reset all user permissions I could find via the terminal.

**Comment by fabricio-fn** [comment]
try restarting your computer to see if that solves it, if the app doesn't open reinstall the app to see if it works

**Reply by agoldh20** [reply]
thanks! I'm sorry I never followed up, I resolved this issue by deleting relevant plist files
